pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -163,6 +163,20 @@ func (s *ServiceImpl) GetNavTree(c *contextmodel.ReqContext, prefs *pref.Prefere
 		return nil, err
 	}
 
+	if s.features.IsEnabled(c.Req.Context(), featuremgmt.FlagPinNavItems) {
+		savedItems := s.buildSavedItemsNavLinks(prefs, treeRoot)
```

Comment:
Wonder if we'd rename `savedItems` to `bookmarks` everywhere in the code to be consistent?